docs/versioned-plugins/inputs/file-v4.4.0.asciidoc

```diff
@@ -9,6 +9,11 @@ START - GENERATED VARIABLES, DO NOT EDIT!
 :release_date: 2021-08-04
 :changelog_url: https://github.com/logstash-plugins/logstash-input-file/blob/v4.4.0/CHANGELOG.md
 :include_path: ../include/6.x
+
+ifeval::["{versioned_docs}"=="true"]
+:branch: 8.2
+:ecs_version: 8.2
+endif::[]
```

Comment:
NOTES: 
- Moved "generated" variables inside "START - GENERATED VARIABLES, DO NOT EDIT!" block
- Manually set `branch` and `ecs_version` to 8.2. 
  - Why? By default, those attributes are set to `current` in the [`index.asciidoc` file](https://github.com/elastic/logstash-docs/blame/versioned_plugin_docs/docs/versioned-plugins/index.asciidoc#L19-L20).